core/src/main/java/tc/oc/pgm/command/ClassCommand.java

```diff
@@ -65,10 +65,12 @@ public void classList(Match match, MatchPlayer player) {
     final PlayerClass currentClass = classes.getSelectedClass(player.getId());
 
     player.sendMessage(
-        LegacyFormatUtils.dashedChatMessage(
-            ChatColor.GOLD + TextTranslations.translate("match.class.title", player.getBukkit()),
-            "-",
-            ChatColor.RED.toString()));
+        text(
+            LegacyFormatUtils.dashedChatMessage(
```

Comment:
Switch this to use [TextFormatter#horizontalLineHeading](https://github.com/PGMDev/PGM/blob/dev/util/src/main/java/tc/oc/pgm/util/text/TextFormatter.java#L110) 😄

Reply:
Yeah I know, I just thought it would make sense to do such a big refactor step by step.

I was thinking TextFormatter needs to be refactored to use `MatchPlayer`s instead anyways

Reply:
Ah just be careful about that, since if an outside plugin wants to use the utils you don’t want them to be forced to use PGM just to get the MatchPlayer.

Reply:
Right, then the bukkit `Player` would also work, yeah? `player.getBukkit()`

Reply:
It depends on the method. For example I use the horizontalLineHeading methods in my requests plugin [here](https://github.com/applenick/Requests/blob/master/src/main/java/tc/oc/occ/Requests/RequestCommand.java#L245). If it were switched to player, then would no longer be able to run the command from the console.

Reply:
Then what about a `Locale` ? This seems to be the only need for the sender

Reply:
I don’t have time atm to look into what the sender is needed for, but awesome yeah if it’s something more general like `Locale` that would be the best path to take 👍